bucket/notepad3.json

```diff
@@ -0,0 +1,47 @@
+{
+    "version": "5.19.108.1602",
+    "description": "Notepad like text editor based on the Scintilla source code.",
+    "homepage": "https://www.rizonesoft.com/downloads/notepad3/",
+    "license": "BSD-3-Clause",
+    "url": "https://github.com/rizonesoft/Notepad3/releases/download/RELEASE_5.19.108.1602/Notepad3_5.19.108.1602.zip",
+    "hash": "df952cbb4f19888732b75dee04c81527ec9a3abdf9e079f12efff1b223afec74",
+    "bin": [
+        "Notepad3.exe",
+        "minipath.exe"
+    ],
+    "shortcuts": [
+        [
+            "Notepad3.exe",
+            "Notepad3"
+        ],
+        [
+            "minipath.exe",
```

Comment:
What is this shortcut for?
This sounds like helper command rather than full gui app

Reply:
I don't know what it is, but it has a gui